src/doc/rustc/src/platform-support/loongarch-linux.md

```diff
@@ -0,0 +1,86 @@
+# loongarch\*-unknown-linux-\*
+
+**Tier: 3**
+
+[LoongArch] is a new RISC ISA developed by Loongson Technology, based in China.
+
+[LoongArch]: https://loongson.github.io/LoongArch-Documentation/README-EN.html
+
+The target name follow this format: `<machine>-<vendor>-<os><fabi_suffix>, where `<machine>` specifies the CPU family/model, `<vendor>` specifies the vendor and `<os>` the operating system name.
+While the integer base ABI is implied by the machine field, the floating point base ABI type is encoded into the os field of the specifier using the string suffix `<fabi-suffix>`.
+
+|    `<fabi-suffix>`     |                           `Description`                            |
+|------------------------|--------------------------------------------------------------------|
+|          f64           | The base ABI use 64-bits FPRs for parameter passing.(lp64d, ilp32d)|
+|          f32           | The base ABI uses 32-bit FPRs for parameter passing. (lp64f ilp32f)|
+|          sf            | The base ABI uses no FPR for parameter passing. (lp64s ilp32s)     |
+
+|`ABI type(Base ABI/ABI extension)`| `C library` | `kernel` |          `target tuple`          |
+|----------------------------------|-------------|----------|----------------------------------|
+|           lp64d/base             |   glibc     |  linux   | loongarch64-unknown-linux-gnuf64 |
+|           lp64f/base             |   glibc     |  linux   | loongarch64-unknown-linux-gnuf32 |
+|           lp64s/base             |   glibc     |  linux   | loongarch64-unknown-linux-gnusf  |
+|           ilp32d/base            |   glibc     |  linux   | loongarch32-unknown-linux-gnuf64 |
+|           ilp32f/base            |   glibc     |  linux   | loongarch32-unknown-linux-gnuf32 |
+|           ilp32s/base            |   glibc     |  linux   | loongarch32-unknown-linux-gnusf  |
+|           lp64d/base             |  musl libc  |  linux   | loongarch64-unknown-linux-muslf64|
+|           lp64f/base             |  musl libc  |  linux   | loongarch64-unknown-linux-muslf32|
+|           lp64s/base             |  musl libc  |  linux   | loongarch64-unknown-linux-muslsf |
+|           ilp32d/base            |  musl libc  |  linux   | loongarch32-unknown-linux-muslf64|
+|           ilp32f/base            |  musl libc  |  linux   | loongarch32-unknown-linux-muslf32|
+|           ilp32s/base            |  musl libc  |  linux   | loongarch32-unknown-linux-muslsf |
+
+## Target maintainers
+
+- zhaixiaojuan, `zhaixiaojuan@loongson.cn`, https://github.com/zhaixiaojuan
```

Comment:
Are there any other people that should be listed here? I see a few different people in this PR 🙂

Reply:
If a third-party maintainer (not affiliated with Loongson) is wanted I guess we can list @xen0n here.

I've ported Rustc to LoongArch64 independently so maybe I can be listed too, but frankly I'm not a Rust expert.